components/nodes/embeds/GoogleForm.js

```diff
@@ -1,15 +1,27 @@
 export default function GoogleForm({ node, amp }) {
+  const id = node.link.split('/')[6];
+
   const el = amp ? (
     <div />
   ) : (
     <iframe
-      src={node.link}
+      src={
+        'https://docs.google.com/forms/d/e/' + id + '/viewform?embedded=true'
+      }
       width="100%"
       height="640px"
       frameBorder="0"
       marginHeight="0"
       marginWidth="0"
     />
   );
-  return el;
+
+  //check that node and node.link arent falsy/undefined log error and don't return anything
+  //check node/link.split also
+
+  if (!node.link || !node || !node.link.split('/')) {
```

Comment:
The check for `!node` should be first because if `node` is falsy (null, undefined, etc.) and you try to access a property of it, you'll see an error.